variables.yml

```diff
@@ -434,12 +434,12 @@ networks:
     chain_id: 1285
     hex_chain_id: '0x505'
     node_directory: /var/lib/moonriver-data
-    parachain_release_tag: v0.45.0
-    parachain_sha256sum: 1f1f3a7ca52f28d9529a4bb01b4ee2ece6c9ed6b771f9a9763750650e7166bfb
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.45.0-3702-bfef
+    parachain_release_tag: v0.46.0
+    parachain_sha256sum: 51d63742aa38a076ab105087c1fab7b82b7f5df368db41130a0e352f240ec0aa
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.46.0-3800-ea4a
     chain_spec: moonriver
     block_explorer: https://moonriver.moonscan.io/
-    spec_version: runtime-3702
+    spec_version: runtime-3800
```

Comment:
The release tags can be updated all at once, but I'd wait on the runtime versions and roll this back for now

Reply:
Sure thing 
The I have upgraded Moonbase only and left the others unchanged per this schedule:

Upgrade Period in Moonbase Alpha
Now - July 09, 2025
Upgrade Period in Moonriver
Now - July 28, 2025
Upgrade Period in Moonbeam
Now - August 11, 2025

Reply:
Thanks!